### ISSUE TITLE
"Rules" option in tsconfig.json not working

### ISSUE THREAD

**Issue by KendrickIW** [palantir/tslint#3068]
### Bug Report

- __TSLint version__: 5.5.0
- __TypeScript version__: 2.3.4
- __Running TSLint via__:  CLI 

#### TypeScript code being linted

with `tslint.json` configuration:

```json
{
    "extends": ["tslint-config-airbnb"],
    "rules": {
        "indent": [true, "spaces",  4]
    }
}
```

#### Actual behavior

Throws errors expecting 2 spaces 

#### Expected behavior

Should not throw errors because I am indenting with 4 spaces
